components/02-molecules/pop-out-image/pop-out-image.stories.js

```diff
@@ -0,0 +1,34 @@
+import popOutImageTwig from './pop-out-image.twig';
+import textFieldTwig from '../text/text-field.twig';
+
+import imageData from '../../01-atoms/images/image/image.yml';
+
+/**
+ * Storybook Definition.
+ */
+export default {
+  title: 'Molecules/Pop Out Image',
+  parameters: {
+    layout: 'fullscreen',
+  },
+  argTypes: {
+    caption: {
+      name: 'Caption',
+      type: 'string',
+      defaultValue: 'This is the caption for the 16:9 image above.',
+    },
+  },
+};
+
+export const PopOutImage = ({ caption }) => `
+  ${textFieldTwig({
+    text_field__content:
+      '<p>The team hypothesizes that cells form distinct types of filaments, activated and inhibited, with unique structures that fine-tune metabolic activity. What’s new about their approach is they will study protein structures by infrared microscopy rather than fluorescence microscopy. They expect these images to reveal a new localized layer of cellular metabolism that could be used to re-engineer metabolic processes. In the future, these complementary time-resolved imaging and cross-linking approaches could be extended to biological studies of macromolecular assemblies and phase-separated membraneless organelles that have been historically difficult to study.</p>',
```

Comment:
seems like this markup should be captured in a yml file instead of here. Clean this file up a bit.

Reply:
Oh, right. This was added to the UI kit, so it's not in the original files. I'll attache a screenshot below for reference.

1) I moved the demo content into a yml file.
2) This is not a variation of the text with image component. Mainly because they have very different content requirements. The text with image will have a separate heading field, and character limits on the text, and a separate link field. This component is an image next to a generic "body" wysiwyg field. (no separate heading/link fields, and unlimited text)
3) There isn't anything in the designs to suggest it should be allowed to float right. @ahitson do you have any opinions on whether this should be allowed to float left or right? Or just left?

<img width="1197" alt="Screen Shot 2022-03-05 at 5 57 09 AM" src="https://user-images.githubusercontent.com/1663810/156881981-775e61f6-0ade-4bb1-b930-f152f22310c5.png">

Reply:
@ModulesUnraveled @ahitson Sounds good that this is a representation of an image within a WYSIWYG. We can scratch the other suggestions I had above. That being said, unless the plan is to remove `align-right` format option from all WYSIWYG profiles we should account for that here as well IMO. 

Otherwise, yml changes look good and I'll approve.

Reply:
One thing to note is that this isn't actually an image inside a wysiwyg. This is in a "Pop-out Image" component that has an image field, and a wysiwyg field (just for text). So there won't be any alignment options needed.